arch/riscv/Kconfig

```diff
@@ -172,6 +172,13 @@ config RISCV_HAS_CLIC
 	help
 	  Does the SOC provide support for a Core-Local Interrupt Controller (CLIC).
 
+config RISCV_SOC_ISR_CHECK
```

Comment:
not to bikeshedding but this config name could be better. We are checking whether the exception is caused by an IRQ (vs EXC). This has nothing to do with ISR really.

Reply:
good point. also investigating CI failures now.

Reply:
renamed to `CONFIG_RISCV_SOC_EXCEPTION_FROM_IRQ`